### PR TITLE
Tests data loaded in real time

### PR DESCRIPTION
**Pull request by Jnegrier**

### CODE REVIEW

File: components/templates/ListOfSuites.tsx [hunk 1]
```diff
@@ -1,200 +1,235 @@
...
+const {loading, noData, testSuites} = getTestSuites(test_run_id, stats.toString());
 
-export const ListOfSuites = function(props: Props) {
-  const { children, stats, darkMode } = props
-  const classes = useStyles(props)
-  const [testInfoSection, setTestInfoSection] = useState(["No test selected"])
-  const [highlightedTest, setHighlightedTest] = useState(0)
+const classes = useStyles(props)
+const [testInfoSection, setTestInfoSection] = useState(["No test selected"])
+const [highlightedTest, setHighlightedTest] = useState(0)
```
Comment (opishcheiko) [inline components/templates/ListOfSuites.tsx:93]: What’s  up with the indentation? Let's run prettifier maybe?

File: data/TestSuites.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        process.env.publicDeltaCore
+    }/api/v1/tests_suites_history/test_status/${
+        stats.toString()
+    }/test_run/${test_run_id}`, fetcher, {refreshInterval: 3000})
+
+    const loading = !data && !error
+    const noData = !data
+
+    return {loading, noData, testSuites: data, mutate};
+}
```
Comment (opishcheiko) [inline data/TestSuites.tsx:16]: nice, why should it be in a separate file? Just in case we want tto use it somewhere else?

File: pages/index.tsx [hunk 3]
```diff
@@ -118,6 +118,21 @@ const useStyles = makeStyles(theme => ({
...
+export interface TestRun {
+  test_run_id: number,
+  launch_id: number,
+  project_id: number,
+  project_name: string,
+  launch: string,
+  test_type: string,
+  start_datetime: string,
+  end_datetime: string,
+  duration: string,
```
Comment (opishcheiko) [inline pages/index.tsx:131]: what happened with duration here? Why string?

Comment (opishcheiko) [reply pages/index.tsx:131]: didn't understand a change in this interface, apart from `data: { url?: string }` to be changed to `test_run_data: {},`

Why does it show like fully new piece of code? I think `launch_name` should stay

**Review by opishcheiko — COMMENTED** [review]
looks good, thanks Juan.
Just was a real pain to review, I don't know what happened with the indentation, and in general, it shows so many changes, that it's hard to track where exaactly was the change

File: components/templates/ListOfSuites.tsx (hunk 1 above)
Comment (Jnegrier) [reply components/templates/ListOfSuites.tsx:93]: @opishcheiko I did a commit where all I did was to run the prettifier D: so the code should be formatted already

File: data/TestSuites.tsx (hunk 2 above)
Comment (Jnegrier) [reply data/TestSuites.tsx:16]: @opishcheiko Yep, one reason is to make it more reusable, the other is to add an abstraction layer for the API calls and have more liberty to modify them

File: components/templates/ListOfSuites.tsx (hunk 1 above)
Comment (opishcheiko) [reply components/templates/ListOfSuites.tsx:93]: doesn't look pretty 😢

File: pages/index.tsx (hunk 3 above)
Comment (Jnegrier) [reply pages/index.tsx:131]: @opishcheiko mmmmmm will take a look into this.... thanks :)

Comment (Jnegrier) [reply pages/index.tsx]: Good catch @opishcheiko it was just `launch` in the Core, so I changed it to `launch_name`, also updated the interface for `duration` and the optional `spectre_test_run_url`

File: components/templates/ListOfSuites.tsx (hunk 1 above)
Comment (Jnegrier) [reply components/templates/ListOfSuites.tsx:93]: 😿

Comment (Jnegrier) [reply components/templates/ListOfSuites.tsx]: I just executed the default VS Code analyzer to format the code, there are some changes but not many, I guess its something we should configure and do at some point

**Comment by Jnegrier** [comment]
@opishcheiko Just added config and formatted with ESLint, looks better now I think 😄

**Review by opishcheiko — APPROVED** [review]

**Pull request merged**